SECURITY.md

```diff
@@ -0,0 +1,30 @@
+# Security Policy
+
+Think you've found a vulnerability in the Uptane Standard? Please review the following details for responsible disclosure.
+
+## Supported Versions
+
+We release updates to the Uptane specification to address security vulnerabilities. Which versions are eligible to receive updates are listed below:
+
+| Version | Supported          |
+| ------- | ------------------ |
+| 1.2.0   | :white_check_mark: |
+| 1.1.0   | :white_check_mark: |
+| 1.0.1   | :white_check_mark: |
+| 1.0.0   | :white_check_mark: |
+
+## Reporting a Vulnerability
+
+Please report (suspected) security vulnerabilities to `uptane-security [AT] googlegroups [DOT] com`. You will receive an acknowledgement from us within 72 hours. Keep in mind this project is a community based project of volunteers and response times could vary. If the issue is confirmed, we will provide a response within 90 days.
```

Comment:
Thanks for the comments @trishankatdatadog . I've opened a PURE to deal with this issue. It's not a problem to leave comments here, but it might be better going forward to post comments at https://github.com/uptane/pures/pull/8 .